airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/check/commands.py

```diff
@@ -30,21 +43,43 @@ async def all_languages(ctx: click.Context, list_errors: bool):
     """
     Run all format checks and fail if any checks fail.
     """
+    commands_to_invoke = [command for command_name, command in check.commands.items() if command_name != ctx.command.name]
+    command_results = await invoke_commands_concurrently(ctx, commands_to_invoke)
+    failure = any([r.status is StepStatus.FAILURE for r in command_results])
+    parent_command = ctx.parent.command
+    logger = logging.getLogger(parent_command.name)
     log_options = LogOptions(
         list_errors=list_errors,
         help_message="Run `airbyte-ci format check all --list-errors` to see detailed error messages for failed checks.",
     )
-    await run_all_subcommands(ctx, log_options)
+    log_command_results(ctx, logger, log_options, command_results)
+    if failure:
+        raise click.Abort()
 
 
 @check.command()
 @pass_pipeline_context
-async def java(pipeline_context: ClickPipelineContext):
+async def python(pipeline_context: ClickPipelineContext) -> CommandResult:
+    """Format python code via black and isort."""
+
+    dagger_client = await pipeline_context.get_dagger_client(pipeline_name="Check python formatting")
+    container = format_python_container(dagger_client)
+    check_commands = [
+        "poetry install --no-root",
+        "poetry run isort --settings-file pyproject.toml --check-only .",
+        "poetry run black --config pyproject.toml --check .",
+    ]
+    return await get_check_command_result(check.commands["python"], check_commands, container)
```

Comment:
You should be able to get the current command from the `pipeline_context`

```
command = pipeline_context._click_context().command
```

Could make this a helper

```
pipeline_context.get_command()
```